app/controllers/application_controller.rb

```diff
@@ -1,16 +1,40 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  around_action :handle_fhir_errors
+
   private
 
-  def create_exporter
+  def exporter
+    @exporter ||= PatientExporter.new(@patient)
+  end
+
+  def find_patient
     @patient = Patient.find(params[:patient_id])
-    @exporter = COVIDHealthCardExporter.new(@patient)
-  rescue ActiveRecord::RecordNotFound => e
-    raise e unless params[:format] == 'fhir_json'
+  end
+
+  def handle_fhir_errors
```

Comment:
This is nice.

Reply:
Thanks! I wasn't sure if there were other errors we should specially handle that wouldn't result in a 500 error.